xml/System.ServiceModel.Channels/ServiceChannelProxy.xml

```diff
@@ -1,1536 +1,1540 @@
+      <Parameters>
+        <Parameter Name="callback" Type="System.AsyncCallback" />
+        <Parameter Name="state" Type="System.Object" />
+      </Parameters>
+      <Docs>
+        <param name="callback">The method that is called when this method completes.</param>
+        <param name="state">Information about the state of the channel.</param>
+        <summary>An asynchronous call to begin using a user interface to obtain credential information.</summary>
+        <returns>The <see cref="T:System.IAsyncResult" /> to use to call back when processing has completed.</returns>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.IClientChannel.DidInteractiveInitialization">
+      <MemberSignature Language="C#" Value="bool System.ServiceModel.IClientChannel.DidInteractiveInitialization { get; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance bool System.ServiceModel.IClientChannel.DidInteractiveInitialization" />
+      <MemberSignature Language="DocId" Value="P:System.ServiceModel.Channels.ServiceChannelProxy.System#ServiceModel#IClientChannel#DidInteractiveInitialization" />
+      <MemberSignature Language="VB.NET" Value=" ReadOnly Property DidInteractiveInitialization As Boolean Implements IClientChannel.DidInteractiveInitialization" />
+      <MemberSignature Language="F#" Value="member this.System.ServiceModel.IClientChannel.DidInteractiveInitialization : bool" Usage="System.ServiceModel.IClientChannel.DidInteractiveInitialization" />
+      <MemberSignature Language="C++ CLI" Value="property bool System::ServiceModel::IClientChannel::DidInteractiveInitialization { bool get(); };" />
+      <MemberType>Property</MemberType>
+      <Implements>
+        <InterfaceMember>P:System.ServiceModel.IClientChannel.DidInteractiveInitialization</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Boolean</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets a value indicating whether a call was done to a user interface to obtain credential information.</summary>
+        <value>
+          <see langword="true" /> if the <see cref="M:System.ServiceModel.IClientChannel.DisplayInitializationUI" /> method was called (or the <see cref="M:System.ServiceModel.IClientChannel.BeginDisplayInitializationUI(System.AsyncCallback,System.Object)" /> and <see cref="M:System.ServiceModel.IClientChannel.EndDisplayInitializationUI(System.IAsyncResult)" /> methods; otherwise, <see langword="false" />.</value>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Channels.ServiceChannelProxy&gt; instance is cast to an &lt;xref:System.ServiceModel.IClientChannel&gt; interface.</remarks>

```

Comment:
```suggestion
        <remarks>This member is an explicit interface member implementation. It can be used only when the <see cref="T:System.ServiceModel.Channels.ServiceChannelProxy"/> instance is cast to an <see cref="T:System.ServiceModel.IClientChannel"/> interface.</remarks>

```